pages/blog/_posts/2020-08-20-building-software-for-a-telecom-core-network.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: blogpost
+permalink: /blog/building-software-for-a-telecom-core-network
+title: Building software for a telecom core network
+date: 2020-08-20
+tags: telecom software-development core-network app-ecosystem
+author: <a href="https://linkedin.com/in/davidaase">David Åse</a>
+---
+
+One of the goals of Working Group Two is to enable operators and third parties to build products and services for the “core network” of the telecom stack. In short, this means providing API access to a subscription’s telecom functionality (messaging, calling, etc). In this article we will show how we built VoiceBox, a Voicemail forwarding application.
+
+## The product
+
+One of the most common complaints people have with their Voicemail service in Europe is that you have to call in to your Voicemail to hear your messages. In order to play the message you have to navigate through a slow voice menu using your dialpad. We believed this would be easy to improve.
+
+Our product idea was simple. When Alice leaves a voicemail message for Bob, Bob receives this message either as an audio file or as a speech-to-text transcript.
+
+<div class="post-images">
+    <img src="/img/blog/building-software-for-a-telecom-core-network/voicebox-splash.png" alt="VoiceBox splash screen">
+    <img src="/img/blog/building-software-for-a-telecom-core-network/voicebox-home.png" alt="VoiceBox home screen">
+</div>
+
+The product idea isn’t what most people would call revolutionary, but almost all of the world's mobile operators have hermetically sealed core networks. This means it would be impossible to build this product without lawyering up and coming to some sort of agreement with one of them. The Working Group Two core network, however, is open and provides API access to Voicemail, MMS and SMS  (among other things), which is just what we need to build our product.
```

Comment:
Could also mention something along the lines of that there are millions/billions of people who have voicemail on their subscription, and if we can make a pretty decent improvement on how they receive them, then that is a lot of time saved in total. Don't know if that makes sense or is worth mentioning, but I thought I'd mention it.

Reply:
I feel like it's a bit of a digression here, but maybe in the conclusion?